ansible/provision.yml

```diff
@@ -27,6 +27,14 @@
     - common
     - mongodb
 
+- name: Message queue
+  user: ubuntu
+  hosts: tag_type_mq:mq
```

Comment:
also the below `tag_type_girder:webserver` should be `webserver`

Other than these 2 changes, LGTM.